collections/_components/quantity-toggle.html

```diff
@@ -1,8 +1,241 @@
 ---
 title: Quantity Toggle
 layout: layout-sidenav
+description: The quantity toggle component is used to increase or decrease an incremental venue, usually best used when the user needs to enter or adjust the quantity of a selected item.
 category: general
-react: true
-img: 'quantity-toggle.png'
+react-link: ?path=/docs/components-quantitytoggle--default-story
+github-link: true
+toc: true
 ---
-{% include page-component/component-header.html %}
\ No newline at end of file
+
+{% capture quantity-toggle-left-disabled %}
+<div class="input-group">
```

Comment:
add variant="quantity-toggle" to div.input-group to remove the default webkit arrows. i just updated the react code today